manifests/implementation/capactio/capact/validation/io/additional-input.yaml

```diff
@@ -0,0 +1,106 @@
+ocfVersion: 0.0.1
```

Comment:
Just a question: do we plan to use this in integration tests, or is it just for demonstration purpose?